css/style.css

```diff
@@ -1525,6 +1547,242 @@ input:focus-visible {
   line-height: 1.33;
 }
 
+.single-article-meta {
+  display: grid;
+  grid-template-columns: 1fr 150px 150px;
+  align-items: center;
+}
+
+.single .about-author {
+  display: flex;
+  align-items: center;
+}
+
+.single .author-avatar {
+  border-radius: 50%;
+  overflow: hidden;
+  width: 48px;
+  height: 48px;
+  margin-right: 13px;
+}
+
+.single .meta-author-circle {
+  width: 48px;
+  height: 48px;
+  background: rgba(15, 54, 46, 0.08);
+  display: flex;
+  border-radius: 50%;
+}
+
+.single .author-info a {
+  display: block;
+  color: var(--dark);
+}
+
+.single .author-info span {
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: 600;
+  line-height: 16px;
+}
+
+.single .meta-comments-count, 
+.single .social {
+  background: #CBEFB6;
+  height: 32px;
+  padding: 4px 16px;
```

Comment:
We could make use of variables of spacing from the root. Shift sequence there and add 16px there. Update rules where shift occurs.